lib/common/template.js

```diff
@@ -108,5 +108,18 @@ function templateServiceFactory(FileLoader, Constants, Promise, _, waterline, Lo
         });
     };
 
+    TemplateService.prototype.unlink = function (name, scope) {
+        scope = scope || 'global';
+        return new Promise(function(resolve, reject) {
```

Comment:
It also could be related to the bluebird/not bluebird promise thing, but I thought we had resolved that (maybe not?).